cmd/download/oc/cmd.go

```diff
@@ -72,73 +69,3 @@ func getPlatform() string {
 	}
 	return runtime.GOOS
 }
-
```

Comment:
Ended up extracting this into it's own command `verify rosa` for some reason prow wasn't able to reach mirror to check it from here which ended up breaking merge flow. Any ideas why? Can we continue like this?